src/lib/utils/thorchain/memo.test.ts

```diff
@@ -0,0 +1,685 @@
+import { describe, expect, it } from 'vitest'
+
+import { assertAndProcessMemo } from './memo' // Update this import path if necessary
```

Comment:
Is this comment needed?

Reply:
de7eb4c4d51018c151cba321e5afcb58919d2235